aiida_vasp/calcs/immigrant.py

```diff
@@ -119,7 +133,8 @@ def get_inputs_from_folder(cls, code, remote_path, **kwargs):
         inputs.code = code
         options = {'max_wallclock_seconds': 1, 'resources': {'num_machines': 1, 'num_mpiprocs_per_machine': 1}}
         inputs.metadata = {'options': options}
```

Comment:
Do we want to open for the possibility to pass `options`? Might be useful down the line if there are some parameters e.g. for instance with storage or something like that which needs to be set.

Reply:
Some scheduler (e.g. sge) requires special `options.resources`.
But I think we don't need to discuss much about `VaspImmigrant` calculation. This is very special, and this speciality is hidden in the level of the workchain.